packages/cactus-plugin-ledger-connector-xdai/src/test/typescript/integration/openapi/openapi-validation.test.ts

```diff
@@ -296,19 +296,19 @@ test(testCase, async (t: Test) => {
         fake: 4,
       };
       await apiClient.invokeContractV1(
-        (parameters as any) as InvokeContractV1Request,
+        (parameters as unknown) as InvokeContractV1Request,
```

Comment:
@micoferdinand98 IIRC this cast can be eliminated entirely. Could you please double check?